### DISCUSSION TITLE
Typo made by copilot?

### DISCUSSION THREAD

**Discussion by SelfMadeSystem** [community/community#11155] [Copilot Conversations] [answered]
When writing a comment for a language I'm developing, copilot learned and understood the language quite well (even though it does not exist), however, it did make a typo which I found quite funny.

![image](https://user-images.githubusercontent.com/38010540/152924599-ab92a972-0ddd-464c-8cf6-b7e4a4748580.png)

Seems like a bug but isn't very important, so I'll put it here. I even deleted and rewrote the comment and it made the same typo.

Edit: no, I didn't have "nothign" in any other part of my code.

**Comment by KTibow** [comment, marked as answer]
I mean... It's [so common it maxes out GitHub's code search](https://github.com/search?q=nothign&type=code) :rofl: 

> Seems like a bug

It's an AI trained on open source :shrug: 

> I even deleted and rewrote the comment and it made the same typo.

Copilot caches suggestions, try using Ctrl+Enter.
